File: FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/DevSettingsActivity.java
/*
 * © 2016-2018 FlowCrypt Limited. Limitations apply. Contact human@flowcrypt.com
 * Contributors: DenBond7
 */

package com.flowcrypt.email.ui.activity;

import android.os.Bundle;
import android.preference.PreferenceActivity;

import com.flowcrypt.email.ui.activity.fragment.preferences.MainDevPreferencesFragment;

/**
 * The developer setting activity. This activity will be used to setup the debug process.
 *
 * @author Denis Bondarenko
 *         Date: 10.07.2017
 *         Time: 10:55
 *         E-mail: DenBond7@gmail.com
 */
public class DevSettingsActivity extends PreferenceActivity {
    @Override
    public void onCreate(final Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getFragmentManager().beginTransaction().replace(android.R.id.content,
                new MainDevPreferencesFragment()).commit();
    }
}

File: FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/DevSettingsActivity.kt
/*
 * © 2016-2019 FlowCrypt Limited. Limitations apply. Contact human@flowcrypt.com
 * Contributors: DenBond7
 */

package com.flowcrypt.email.ui.activity;

import android.view.View;

import com.flowcrypt.email.R;
import com.flowcrypt.email.ui.activity.base.BaseBackStackActivity;

/**
 * The developer setting activity. This activity will be used to setup the debug process.
 *
 * @author Denis Bondarenko
 * Date: 10.07.2017
 * Time: 10:55
 * E-mail: DenBond7@gmail.com
 */
public class DevSettingsActivity extends BaseBackStackActivity {
  @Override
  public int getContentViewResourceId() {
    return R.layout.activity_dev_settings;
  }

  @Override
  public View getRootView() {
    return null;
  }
}
